src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -85,6 +85,41 @@ public override string ToString()
     }
 }";
 
+        [Fact]
+        [WorkItem(25529, "https://github.com/dotnet/roslyn/issues/25529")]
+        public void UnassignedNonNullFieldsUnreachable()
```

Comment:
>UnassignedNonNullFieldsUnreachable [](start = 20, length = 34)

Consider moving to `UninitializedNonNullableFieldTests`.

Reply:
Also please add tests that have return statements inside the constructor (at the end, and inside of an if statement)

---
In reply to: [267909107](https://github.com/dotnet/roslyn/pull/34316#discussion_r267909107) [](ancestors = 267909107)